src/Cli/Microsoft.TemplateEngine.Cli/Commands/CliPathInfo.cs

```diff
@@ -42,6 +42,7 @@ private static string GetUserProfileDir(IEnvironment environment) => environment
             RuntimeInformation.IsOSPlatform(OSPlatform.Windows)
                 ? "USERPROFILE"
                 : "HOME")
+            ?? Environment.GetFolderPath(Environment.SpecialFolder.UserProfile)
```

Comment:
Would it make sense to just call GetDotnetHomePath here? I believe template engine cli already has a reference to Microsoft.DotNet.Configurer so it should be available.